core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -161,10 +161,14 @@ class AdminClient(val time: Time,
     if (group.protocolType != ConsumerProtocol.PROTOCOL_TYPE)
       throw new IllegalArgumentException(s"Group ${groupId} with protocol type '${group.protocolType}' is not a valid consumer group")
 
-    group.members.map {
-      case member =>
-        val assignment = ConsumerProtocol.deserializeAssignment(ByteBuffer.wrap(member.assignment))
-        new ConsumerSummary(member.memberId, member.clientId, member.clientHost, assignment.partitions().asScala.toList)
+    if (group.state == "Stable") {
+      group.members.map {
+        case member =>
```

Comment:
This is identical to https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/admin/AdminClient.scala#L141. We should have such checks be taken care by some tools, so that it is consistent. Adopting your suggestion, and applying it to https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/admin/AdminClient.scala#L141 as well.